src/IJulia/setup.jl

```diff
@@ -92,36 +93,35 @@ mimewritable(m::MIME, s::Signal) =
     mimewritable(m, s.value)
 
 # fixes ambiguity warnings
-function writemime(io:: IO, m::MIME"text/plain", s::Signal)
-    writemime(io, m, s.value)
+@compat function Base.show(io::IO, m::MIME"text/plain", s::Signal)
+    Base.show(io, m, s.value)
```

Comment:
Ah, I see! Thanks :-)

On 13-Aug-2016 4:44 PM, "Yichao Yu" <notifications@github.com> wrote:

> In src/IJulia/setup.jl
> <https://github.com/JuliaLang/Interact.jl/pull/100#discussion_r74685667>:
>
> > @@ -92,36 +93,35 @@ mimewritable(m::MIME, s::Signal) =
> >      mimewritable(m, s.value)
> >
> >  # fixes ambiguity warnings
> > -function writemime(io:: IO, m::MIME"text/plain", s::Signal)
> > -    writemime(io, m, s.value)
> > +@compat function Base.show(io::IO, m::MIME"text/plain", s::Signal)
> > +    Base.show(io, m, s.value)
>
> yuyichao% JULIA_LOAD_PATH=/usr/share/julia/site/v0.6 ~/projects/julia/tests/release-0.4/julia
>                _
>    _       _ _(_)_     |  A fresh approach to technical computing
>   (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
>    _ _   _| |_  __ _   |  Type "?help" for help.
>   | | | | | | |/ _` |  |  | | |_| | | | (_| |  |  Version 0.4.6-pre+18 (2016-04-01 03:37 UTC) _/ |\__'_|_|_|\__'_|  |  Commit 198644a* (134 days old release-0.4)|__/                   |  x86_64-unknown-linux-gnujulia> using CompatINFO: Recompiling stale cache file /home/yuyichao/.julia/lib/v0.4/Compat.ji for module Compat.julia> macroexpand(:(@compat function Base.show(a, b, c) Base.show(a, b.d, c) end)):(function Base.writemime(a,b,c) # none, line 1:        Base.writemime(a,b.d,c)    end)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/Interact.jl/pull/100/files/f4c90e5132919f374ca06366be12841437332d16#r74685667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABlPFhPgFA3hzeHaUUcHMyeBz3SolfEks5qfacUgaJpZM4JCFND>
> .
>